tools/code_format/config.yaml

```diff
@@ -381,6 +381,7 @@ replacements:
     # Well-known types should be referenced from the ProtobufWkt namespace.
     "Protobuf::Any": "ProtobufWkt::Any"
     "Protobuf::Empty": "ProtobufWkt::Empty"
+    "Protobuf::Field": "ProtobufWkt::Field"
```

Comment:
This is problematic because it apparently matches `Protobuf::FieldDescriptor` and `Protobuf::FieldMask` which would not be correct to switch to `ProtobufWkt` (and insists that you update all of them all at once to pass CI). This seems like an error in the linter behavior rather than an error in your change.

The *ideal* action here would be to fix that, but if that's not in your wheelhouse I would be on board with just the one-time fix in this PR (reverting this file), and filing an issue for someone more familiar with the linter to add `Protobuf::Field` here and make it work correctly.

Reply:
Correction, `FieldMask` *should* be here as well, but `FieldDescriptor` should not.

Reply:
im working on the linter atm so can possibly take a look

Reply:
Yes, this config change is incorrect. The linter currently uses python's `str.replace` to [substitute](https://github.com/envoyproxy/envoy/blob/main/tools/code_format/check_format.py#L503). A regex base solution would work but might make the code formatter very slow.  

> one-time fix in this PR (reverting this file),

I'll revert the code_format changes & make this PR a one-time fix. Will take a shot at updating the linter in a different PR.